mobile/native/src/db/mod.rs

```diff
@@ -51,3 +54,32 @@ pub fn update_last_login() -> Result<api::LastLogin> {
     let last_login = models::LastLogin::update_last_login(now, &mut db)?.into();
     Ok(last_login)
 }
+
+pub fn insert_order(order: trade::order::Order) -> Result<trade::order::Order> {
+    let mut db = connection()?;
+    let order = Order::insert(order.into(), &mut db)?;
```

Comment:
In my PR #237 I decided to change this to free-floating functions move these functions into a separate module instead, e.g. `orders::insert`